.bazelrc

```diff
@@ -1,71 +1,102 @@
+################################################################
+# Config from TensorFlow's .bazelrc.                           #
+#                                                              #
+# This is copied directly, with some irrelevant parts removed. #
+################################################################
+
+# Default build options. These are applied first and unconditionally.
+
+# The following line opts in to modular op registration support by default.
+build --define framework_shared_object=true
+
+# For workaround https://github.com/bazelbuild/bazel/issues/8772 with Bazel >= 0.29.1
+build --java_toolchain=@tf_toolchains//toolchains/java:tf_java_toolchain
+build --host_java_toolchain=@tf_toolchains//toolchains/java:tf_java_toolchain
```

Comment:
Is this a workaround we actually need, or is it just included for completeness and ease of future upgrades?

Reply:
I just included it for completeness but happy to remove it if you prefer

Reply:
Nah that's fine we can keep them, I was just wondering.